moduleroot/.travis.yml.erb

```diff
@@ -4,9 +4,9 @@
 <% if @configs['test_all_rvms'] -%>
 rvm:
   - 2.1.9
-  - 2.2.6
   - 2.3.0
   - 2.4.1
+  - 2.5.1
```

Comment:
Adding this here and then excluding it in the matrix is redundant.

Reply:
changed.